### DISCUSSION TITLE
Add Ctrl-[ to close NNN

### DISCUSSION THREAD

**Discussion by arktnld** [mcchrish/nnn.vim#158] [Ideas]
I use this command a lot and I miss it in the plugin, this is a standard command in vim

**Comment by mcchrish** [comment]
Sounds good. It should be an easy addition. Would you like to create a PR?

**Comment by mcchrish** [comment]
You can also map it on your own: https://github.com/mcchrish/nnn.vim/discussions/118#discussioncomment-1439776
